src/ui.css

```
@@ -1,499 +1,500 @@
+.ytaf-notification-container.theme-classic-red .message {
+  background: #1a1a1a;
+  border-color: #3f3f3f;
+  border-left-color: #ff0000;
+}
+
+.ytaf-notification-container .message-hidden {
+  opacity: 0;
+  transform: translateX(1.85vh);
+}
+
+<<<<<<< HEAD
```

Stylelint (17.14.0)

[error] Expected "HEAD" to be "head" (selector-type-case)

(selector-type-case)